### PR TITLE
Fetch CoraLibre-android-sdk from Jitpack

### LINKED ISSUE #5: Replace git submodule with jitpack package
Replace SDK submodule with jitpack package.

Depends on https://github.com/CoraLibre/CoraLibre-android-sdk/issues/36

### PR DESCRIPTION
**Pull request by Stypox**
This PR removes the CoraLibre-android-sdk submodule, and uses the library build provided by JitPack. I also had to add JitPack's maven repo as a dependency source: `maven { url 'https://jitpack.io' }`. The app builds fine and all tests pass. Since JitPack support has not yet been merged on the sdk side, for now I used my fork, which should be replaced with our repo before merging.

The version of the sdk downloaded from JitPack is currently identified by the (full) hash of the wanted commit. In my opinion this is the optimal way. With NewPipe at the beginning we used named versions (i.e. NewPipe 0.16.2 used NewPipeExtractor version `v0.16.2`), but that implied having to change the version every time before release. Also, most of the times we ended up using commit hashes anyway (e.g. when a change in the app required a new version of the extractor, but before the release). So at some point we stopped using version names alltogether, and I think this is the best way to go. 
  
Depends on and should be merged together with CoraLibre/CoraLibre-android-sdk#42
Fixes #5

### CODE REVIEW

**Review by theScrabi — CHANGES_REQUESTED** [review]
LFTM :)
However this PR should be blocked until https://github.com/CoraLibre/CoraLibre-android-sdk/issues/36 is through.

File: Corona-Warn-App/build.gradle
Comment (theScrabi) [inline Corona-Warn-App/build.gradle]: Please update to the latest commit: 4d4cc65a6ead6437863c5d4adad7e0aa67da54e4

**Review by theScrabi — DISMISSED** [review]

**Comment by Stypox** [comment]
Now that everything was merged the JitPack library can't be resolved anymore for some reason. But everything builds fine on JitPack's side: https://jitpack.io/#CoraLibre/CoraLibre-android-sdk
I'll look into this in the next days

File: Corona-Warn-App/build.gradle
Comment (BjoernPetersen) [inline Corona-Warn-App/build.gradle]: This targets the wrong project and an outdated tag.
```suggestion
    implementation 'com.github.CoraLibre:CoraLibre-android-sdk:4d4cc65a6ead6437863c5d4adad7e0aa67da54e4'
```

**Review by BjoernPetersen — CHANGES_REQUESTED** [review]

**Comment by BjoernPetersen** [comment]
I created a PR on the SDK fixing the JitPack build: https://github.com/CoraLibre/CoraLibre-android-sdk/pull/44

When that PR is merged, this PR needs to be updated to the newest commit and can be merged, provided it works correctly then.

**Comment by Stypox** [comment]
Done, now commit https://github.com/CoraLibre/CoraLibre-android-sdk/commit/592ec0093d08a0fb276150c45ecfe309f4a694d9 is referenced and the build succeeds. Thank you @ BjoernPetersen for figuring out how to have the library collect the artifacts, idk why for me it worked when I created the PR, but then it didn't work anymore after merging ;-)

**Review by BjoernPetersen — APPROVED** [review]

**Pull request merged**
